dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -115,6 +128,73 @@ def DALIIterator():
 def DALIRawIterator():
     return _dali_tf
 
+def get_tf_minor_version():
+  return tf.__version__.split('.')[1]
+
+
+class DALIDatasetV2(dataset_ops.DatasetSource):
```

Comment:
how about having two separate modules (`v1.DALIDataset`, and `v2.DALIDataset`)

Reply:
This is sort of an idiom in the Tensorflow codebase (e.x. [`RandomDataset`](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/data/experimental/ops/random_ops.py#L32)) and it is there for compatibility reasons. When we will start to support TF 2.0 we will need it.
We could call them `*Wrapper` and `*Impl` for now but I wanted to do it like they do.